src/firebase/api/gcp-utils.js

```diff
@@ -1,74 +1,82 @@
-import { storage, pubsub as gcpPubsub } from '../firebase.js';
-import { pubsub } from '../constants.js';
+import { app, pubsub as gcpPubsub } from '../firebase.js';
+import { pubsub, storageBuckets } from '../constants.js';
 
 /**
  * Upload an image to the specified bucket
  * @param {string} userId userId
  * @param {string} experimentId experimentId
  * @param {Blob} image image file to be uploaded
- * @param {string} bucket destination bucket
  *
  * @return {JSON}
  */
 export const uploadImageToStorage = async (
-  userId, experimentId, image, bucket) => {
+  userId, experimentId, image) => {
   const imagePath = `${userId}-${experimentId}/${image.name}`;
 
-  const storageRef = storage.ref();
-  const rawImageBucketRef = storageRef.child(bucket);
+  const rawImageBucketRef = app.storage(storageBuckets.SIE_RAW_IMGS).ref();
   const newImageRef = rawImageBucketRef.child(imagePath);
-  newImageRef.put(image).then((snapshot) => {
+  return await newImageRef.put(image).then(() => {
     return {
-      status: '201',
-      error: null,
+      status: 201,
```

Comment:
do you mind replacing all these ints with StatusCodes.CREATED, etc.?

Reply:
Will update